test/nlcManager.test.js

```diff
@@ -52,6 +52,13 @@ describe('Test the NLCManager library', function(){
 		});
 	});
 
+	it('should successfully get the status of most recent classifier', function(done){
+		watson_nlc.classifierStatus().then(function(result){
+			expect(result.status).to.be.equal('Available');
+			done();
+		});
+	});
+
```

Comment:
@jlpadilla 

This test should covers the `doNotTrain` route since no `classifier_id` was provided in `.classifierStatus()`.